appIcons.js

```diff
@@ -1233,8 +1233,11 @@ function getInterestingWindows(windows, monitorIndex) {
         const showUrgent = settings.get_boolean('workspace-agnostic-urgent-windows');
         const activeWorkspace = global.workspace_manager.get_active_workspace_index();
         windows = windows.filter(function(w) {
-            return w.get_workspace().index() == activeWorkspace
-                || (showUrgent && (w.urgent || w.demandsAttention || w._manualUrgency));
+            // See GJS docs : Meta/Window
+            const inWorkspace = w.get_workspace().index() == activeWorkspace;
+            const isUrgent = w.urgent || w.demandsAttention || w._manualUrgency;
+            const isNotInTaskbar = w.is_skip_taskbar();
```

Comment:
I was thinking to include this for some time now too... I've been always wondering if this would break the case of showing the windows into the previews-menu though.

In fact while the windows may not be to be shown in the lists, *maybe* we may want to show them in the previews overview... But I'm not either sure about :)